CHANGELOG.md

```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 This changelog format was introduced in NAV 5.4.0. Older changelogs can be
 found in the [HISTORY](HISTORY) file.
 
+## [Unreleased]
+
+
+
+## bugfix/room-search-related-names
+
+### Fixed
+
+#### User-visible fixes
+- Fix backwards relation references in room search ([#2595](https://github.com/Uninett/nav/pull/2595))
```

Comment:
Not entirely sure I agree this is a user-visible fix, unless there are users that actually keep track of the master branch (i.e. the bug you're fixing has never made it into a release)

Reply:
At best, I would include the PR in the list of PRs mentioned for the (still non-existant) log entry for "renamed relations". Like the the one used for [NAV 5.6.0](https://github.com/Uninett/nav/releases/tag/5.6.0):
> Updated/added explicit relation names to various ORM models (2539, 2540, 2541, 2542)